src/ngraph/pass/manager.hpp

```diff
@@ -54,6 +54,14 @@ class ngraph::pass::Manager
         }
     }
 
+    void set_ops_to_details(
+        std::unordered_map<std::type_index,
+                           std::function<void(std::shared_ptr<Node>, std::stringstream& ss)>>*
```

Comment:
If you just made this `Node*` instead of `std::shared_ptr<Node>` it would be smaller and clearer. Or maybe a Node&
Also use and ostream& just to be a little more generic.

Reply:
Would this look better from a user point of view if it were an initializer list?